src/DssSpell.t.sol

```diff
@@ -399,325 +396,22 @@ contract DssSpellTest is DssSpellTestBase {
-
-    function testFlashLegacy() public {
-        FlashAbstract flash = FlashAbstract(addr.addr("MCD_FLASH_LEGACY"));
-        assertEq(flash.max(), 500_000_000 * WAD);
-
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-
-        uint256 vowDai = vat.dai(address(vow));
-
-        uint256 amount = 1 * MILLION;
-        assertEq(flash.toll(), 0);
-        assertEq(flash.flashFee(address(dai), amount * WAD), 0);
-        assertEq(flash.max(), 250_000_000 * WAD);
-        assertEq(flash.flashFee(address(dai), amount * WAD), 0); // 0 DAI fee on a 1M loan (on any actually)
-        flash.flashLoan(address(this), address(dai), amount * WAD, "");
-        flash.vatDaiFlashLoan(address(this), amount * RAD, "");
-        assertEq(vat.sin(address(flash)), 0);
-        assertEq(vat.dai(address(flash)), 0);
-        flash.accrue();
-        assertEq(vat.dai(address(flash)), 0);
-        assertEq(vat.dai(address(vow)), vowDai);
-    }
-
-    function testFlash() public {
-        FlashAbstract flash = FlashAbstract(addr.addr("MCD_FLASH"));
-
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-
-        uint256 amount = 1 * MILLION;
-        assertEq(flash.flashFee(address(dai), amount * WAD), 0);
-        assertEq(flash.max(), 250_000_000 * WAD);
-        assertEq(flash.flashFee(address(dai), amount * WAD), 0); // 0 DAI fee on a 1M loan (on any actually)
-        flash.flashLoan(address(this), address(dai), amount * WAD, "");
-        flash.vatDaiFlashLoan(address(this), amount * RAD, "");
-        assertEq(vat.sin(address(flash)), 0);
-        assertEq(vat.dai(address(flash)), 0);
-    }
-
-    function onFlashLoan(
-        address initiator,
-        address token,
-        uint256 amount,
-        uint256 fee,
-        bytes calldata
-    ) external returns (bytes32) {
-        assertEq(initiator, address(this));
-        assertEq(token, address(dai));
-        assertEq(amount, 1 * MILLION * WAD);
-        assertEq(fee, 0);
-
-        dai.approve(msg.sender, amount);
-
-        return keccak256("ERC3156FlashBorrower.onFlashLoan");
-    }
-
-    function onVatDaiFlashLoan(
-        address initiator,
-        uint256 amount,
-        uint256 fee,
-        bytes calldata
-    ) external returns (bytes32) {
-        assertEq(initiator, address(this));
-        assertEq(amount, 1 * MILLION * RAD);
-        assertEq(fee, 0);
-
-        vat.move(address(this), msg.sender, amount);
-
-        return keccak256("VatDaiFlashBorrower.onVatDaiFlashLoan");
-    }
-
     function testMKRPayment() public {
         uint256 prevMkrPause = gov.balanceOf(address(pauseProxy));
-        uint256 prevMkrSES = gov.balanceOf(wallets.addr("SES_WALLET"));
+        uint256 prevMkrSAS = gov.balanceOf(wallets.addr("SIDESTREAM_WALLET"));
+        uint256 prevMkrDUX = gov.balanceOf(wallets.addr("DUX_WALLET"));
+
+        uint256 amountSAS = 243.7953 ether;
```

Comment:
```suggestion
        uint256 amountSAS =  243795300000000000000;
```

Would suggest testing against a different format than used in the spell just to assuage concerns about the keyword.